src/test/resources/mappings/error.json

```diff
@@ -5,7 +5,7 @@
         "method": "POST",
         "url": "/users?lang=de",
         "bodyPatterns" : [ {
-          "equalToJson" : { "customFields" : { }}
+          "equalToJson" : { "customFields" : { }, "proxyFor" : [], "departments" :  []}
```

Comment:
The matcher got more strict for the `createUser_shouldReturn500Error_whenRequestBodyIsEmpty` test after the upgrades.